torchdynamo_poc/TorchDynamo Demo.ipynb

```diff
@@ -219,7 +253,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "id": "964ddfde",
    "metadata": {},
    "outputs": [
```

Comment:
It might be worth adding immediately after this something that demonstrates that the print statement is stripped from the trace and/or state it explicitly (e.g. "Note the lack of the print statement in the generated graph.").

Reply:
Done!